core/js/systemtags/systemtagsinputfield.js

```diff
@@ -259,6 +261,17 @@
 		 * @param {Object} query select2 query object
 		 */
 		_queryTagsAutocomplete: function(query) {
+			var termErrors = false;
+
+			if(query.term.indexOf('@') !== -1) {
+				termErrors = true;
+				this.$errorField.text(t('systemtags', 'This looks like an email address or federated cloud ID. Are you sure you want to use a tag?')).removeClass('hidden');
```

Comment:
```suggestion
				this.$errorField.text(t('systemtags', 'This looks like an email address or federated cloud ID. Are you sure you want to use this tag name?')).removeClass('hidden');
```
You are not arguing about using a tag in general but about the particular name used